global-script.lua

```diff
@@ -2152,12 +2152,17 @@ function resetPiece(object, rotation, offset)
 end
 function handlePlayer(color, data)
     local zone = getObjectFromGUID(elementScanZones[color])
+    local spiritBoardTopEdge = getObjectFromGUID(elementScanZones["Red"]).getPosition().z
```

Comment:
why not just use `zone` here?

Reply:
This is copy paste from an old version of the code (#3) that did not have that variable. Fixed.